src/http2.ts

```diff
@@ -38,7 +38,7 @@ const DEBUG = !!process.env.HTTP2_DEBUG;
  */
 export interface SessionData {
   session: http2.ClientHttp2Session;
-  timeoutHandle?: NodeJS.Timer;
+  timeoutHandle?: NodeJS.Timeout;
```

Comment:
Typically, this would be considered a breaking change, however the existing type in itself is a bug and must be fixed as it is blocking all PRs in this library.